src/main/java/frc/robot/commands/FeederCommand.java

```diff
@@ -5,36 +5,38 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.OperatorBoard.OperatorBoardButton;
 import frc.robot.subsystems.ShooterSubsystem;
 
 /** Command to shoot one ball. */
```

Comment:
javadoc should be updated since shootoneballcommand was replaced with feedercommand